db/migrate/20191028115549_create_application_qualification.rb

```diff
@@ -0,0 +1,15 @@
+class CreateApplicationQualification < ActiveRecord::Migration[6.0]
+  def change
+    create_table :application_qualifications do |t|
+      t.references :application_form, null: false, foreign_key: { on_delete: :cascade }
+
+      t.string :type, null: false
+      t.string :qualification_type, null: false
```

Comment:
How does this work with the form for GCSEs?

<img width="734" alt="Screenshot 2019-10-28 at 14 30 28" src="https://user-images.githubusercontent.com/642279/67687029-8ba50c80-f98f-11e9-82f2-92339f37ad6e.png">

Reply:
We've added `awarding_body` now.